packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -48,6 +48,17 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   onChanged?: (option?: IComboBoxOption, index?: number, value?: string) => void;
 
+  /**
+   * Callback issued when the user changes the pending value index in ComboBox
+   * This includes hovering or keyboarding to a menu item
+   */
+  onPreviewExecute?: (option?: IComboBoxOption, index?: number, value?: string) => void;
+
+  /**
+   * Callback issued when user leaves the menu item in the ComboBox menu
+   */
+  onRevertPreviewExecute?: () => void;

```

Comment:
Do we actually need a revert callback? There's either going to be a new pending value or it's there's no pending value. I think onPendingValueChanged could handle both cases